code/modules/mob/living/silicon/robot/life.dm

```diff
@@ -14,6 +14,13 @@
 	if(client)
 		handle_regular_hud_updates()
 		update_items()
+	if(stat)
+		cut_overlay(eye_overlay)
+		has_cut_eye_overlay = TRUE
+	else if(has_cut_eye_overlay)
+		eye_overlay = cached_eye_overlays[a_intent]
+		add_overlay(eye_overlay)
+		has_cut_eye_overlay = null
```

Comment:
Why null and not FALSE?

Reply:
afaik null takes up slightly less space than 0, and there's real reason for it to be 0 instead of null